Documentation/gettingstarted/dns.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************************************
+Getting Started Using DNS-Based Policies
+****************************************
+
+This document serves as an introduction for using Cilium to enforce DNS-based
+security policies for Kubernetes pods.
+
+.. include:: gsg_intro.rst
+.. include:: minikube_intro.rst
+.. include:: cilium_install.rst
+
+Step 2: Deploy the Demo Application
+===================================
+
+DNS-based policies are very useful for controlling access to services running outside the Kubernetes cluster. Whether it is cloud services such as S3, DynamoDB, RDS, etc. or private VPC services outside the cluster, DNS provides persistent identity for these external services. CIDR or IP-based policies are cumbersome and hard to maintain as the IPs associated with external services can change frequently. The Cilium DNS-based policies provide an easy mechanism to specify access control while leaving the harder aspects of resolving and enforcing IP-based filters to Cilium. 
+
+In this guide we will learn about:  
+ 
+- Controlling egress access to services outside the cluster using DNS-based policies
+- Using patterns (or wildcards) to whitelist a subset of DNS domains
+- Combining DNS and port rules for restricting access to external service  
+
+Step 3: Create Example App
+==========================
+
+In line with our Star Wars theme examples, we will use simple scenario with two pods, ``mediabot`` and ``spaceship``. The ``mediabot`` pod needs access to Twitter API services for managing the empire's tweets. However, the ``spaceship`` pods shouldn't have any external service access for obvious reasons. 
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/dns-sw-app.yaml
```

Comment:
There is nothing minikube specific about this demo app deployment. This guide should not give the impression that it is minikube specific in any way.

Reply:
moved all the app and policies to new folder `examples/kubernetes-dns/`